nrf_modem/doc/gnss_interface.rst

```diff
@@ -446,7 +446,7 @@ In cases where GNSS is not running continuously, it may be beneficial to change
 
 .. note::
 
-   1PPS accuracy is not guaranteed when LTE is active.
+   1PPS feature must not be used when LTE is enabled.
```

Comment:
```suggestion
   Do not use the 1PPS feature when LTE is enabled.
```

Reply:
I would have to disagree. In fact, "Do not use..." is actually a lot more aggressive way of putting this, but up to you. The original is not misleading by any means.

Reply:
Maybe it's just my background with IETF RFCs, where "must not" means "absolute prohibition" (https://datatracker.ietf.org/doc/html/rfc2119). 😄 

Ok, let's go with your suggestion.